extra/release_tool.py

```diff
@@ -2586,6 +2586,82 @@ def determine_version_to_include_in_release(state, repo):
     return True
 
 
+def run_and_combine_statistics_and_changelog(
+    workdir, single_repo, from_version, to_version, output_file
+):
+    """Generate statistics and changelogs and combine them in a single file"""
+
+    release_name = "%s %s" % (
+        "Mender" if not single_repo else os.path.basename(workdir),
+        to_version,
+    )
+    release_header = "## %s\n\n_Released xx.xx.xxxx_\n\n" % release_name
```

Comment:
Suggestion, why not get the `xx.xx.xxxx` version from the git tag?

Reply:
@oleorhagen What if we use today's date? It will be right most of the times... Making it getting it from the repository is not trivial.

Reply:
Hmm, using today's will not do, I'm afraid, say I tagged already, but doing the changelogs a few days later, which is not that uncommon, at least not for me.
But you can just get the tag from the integration repository can you not? Like we know which one it is

Reply:
Done :muscle:

Reply:
:muscle: